python_src/src/lamp_py/tableau/jobs/rt_rail.py

```diff
@@ -178,17 +178,23 @@ def update_parquet(self, db_manager: DatabaseManager) -> bool:
             f" AND vt.service_date >= {max_start_date.strftime('%Y%m%d')} ",
         )
 
-        db_parquet_path = "/tmp/db_local.parquet"
         db_manager.write_to_parquet(
             select_query=sa.text(update_query),
-            write_path=db_parquet_path,
+            write_path=self.db_parquet_path,
```

Comment:
should batch size be included in this function call?

Reply:
We're ok with the default batch size `1024*1024` so I just left it out.